app/3.0/start/samples/custom-element/index.html

```diff
@@ -1,10 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <script src="https://polygit.org/components/webcomponentsjs/webcomponents-loader.js"></script>
-    <link rel="import" href="custom-element.html">
+    <script src="https://polygit.org/webcomponentsjs/components/webcomponentsjs/webcomponents-loader.js"></script>
```

Comment:
wc-loader URL should not change.

JS files (like custom-element.js) might not be copied to dist/, which may be why the demos aren't working (I see a bunch of 404s)

Reply:
oh yeah that makes sense (both things). Thank you! Fixing the wc-loader links now.

@arthurevans @keanulee what would be the correct process to get the samples into the distribution properly? I tried pasting one into the appropriate location in the dist/ folder but the sample still didn't render, idk

Reply:
Add to `gulp.task('copy', ...)`, something like:

```js
  const jsSamples = gulp.src([
      'app/**/*.js',
      '!app/{bower_components,elements}/**',
      '!app/3.0/samples/homepage/**', // excluded since homepage samples are put through polymer build - see demo1 below
      '!app/js/**/*.js'
    ], {base: 'app/'})
    .pipe(gulp.dest('dist'));

  return merge(app, docs, /* ... */, jsSamples);
```

Reply:
Hmm, seems like there's more config to do. I'll take a look a push a commit when I figure it out

Reply:
<3 <3 <3

Reply:
Give that a try. The editable-name sample is broken because unpkg won't redirect deps to pre-release versions (but you probably already knew that).

Reply:
omg, it's doing the thing

Thank you!!!!